### PR TITLE
Document lastScan field in /rest/stats/folder API endpoint

### PR DESCRIPTION
**Pull request by majedev**
As per changes proposed in [this PR](https://github.com/syncthing/syncthing/pull/3250).

### CODE REVIEW

File: rest/stats-folder-get.rst [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-Returns general statistics about folders. Currently, only contains the
-last synced file.
+Returns general statistics about folders. Currently contains the
+last scan time and the last synced file.
 
 .. code-block:: bash
 
     $ curl -s http://localhost:8384/rest/stats/folder | json
     {
+      "lastScan": "2016-06-02T13:28:01.288181412-04:00",
```
Comment (calmh) [inline rest/stats-folder-get.rst]: But not outside the folder ID right?

Comment (majedev) [inline rest/stats-folder-get.rst]: Ugh, right. I need coffee.

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 88f88a948d85f71443eda97aa00e795144afaa2c. Thanks, @majedev!

**Pull request closed without merging**